pkg/cmd/target/control-plane.go

```diff
@@ -0,0 +1,36 @@
+/*
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Projecter contributors
```

Comment:
2022 (also with the other files)

Reply:
Yes, we can update all files when we have less PRs open. WDYT?

Reply:
sorry I meant:
2022 (also with the other newly introduced files)

Reply:
done